specs/simple-serialize.md

```diff
@@ -177,38 +175,36 @@ Note that deserialization requires hardening against invalid inputs. A non-exhau
 
 We first define helper functions:
 
+* `chunk_count(type)`: calculate the amount of leafs for merkleization of the type.
+   * all basic types: `1`
+   * bitlists and bitvectors: `(N + 255) // 256` (dividing by chunk size, rounding up)
+   * lists and vectors of basic types: `N * item_length(elem_type) + 31) // 32` (dividing by chunk size, rounding up)
```

Comment:
`item_length` is not defined in the spec and there looks to be an un-opened trailing paren after `31`

Reply:
defined `size_of` in 1 line, for basic types. See d9fd1d3